README.md

```diff
@@ -0,0 +1,27 @@
+Project Name: Hello Microverse
+
+In this project we learn about:
+-linterns
+-gitflow and specifically pull requests
+-documenting work
+
+
+Built With: html and css
+
+Live Demo Link: http://127.0.0.1:5500/index.html
```

Comment:
- [ ] It seems that your live demo link is broken since you have used the local link, for you to have a demo link the project needs to be deployed first. feel free to remove the demo link section if you have not deployed the project yet 😉 
- [ ] I think it could be a good idea to keep the structure of the README file like the one we have in the Microverse README template, you can find it [here](https://github.com/microverseinc/readme-template/blob/master/README.md). This will keep the README professional 👍

Reply:
Thanks, I did not know about copying the "RAW" of the template. Someone in my Stand up group told me.